api/helpers.js

```diff
@@ -1,3 +1,8 @@
 export function getConfig (content) {
   return content.find(({ _type, __path__ }) => _type === 'config' || __path__.endsWith('config.json'))
 }
+
+export function getGlobals (content) {
+  const course = content.find(({ _type, __path__ }) => _type === 'course' || __path__.endsWith('course.json'));
+  return course?._globals;
+}
```

Comment:
This function is mostly, getCourse, not getGlobals.

```suggestion
export function getCourse(content) {
  return content.find(({ _type }) => _type === 'course');
}
```
The course object, unlike config, has always had a _type, regardless of the environment, so find by __path__ isn't necessary.

Then use `getCourse()._globals` to get the globals.